src/main/java/com/powsybl/openloadflow/equations/EquationEventType.java

```diff
@@ -12,7 +12,6 @@
 public enum EquationEventType {
     EQUATION_CREATED,
     EQUATION_REMOVED,
-    EQUATION_UPDATED,
```

Comment:
You have removed that type because now you want to introduce the equation term granularity ?

Reply:
Yes